parent/pom.xml

```diff
@@ -243,11 +243,13 @@
 					<!-- set useUIThread=false for swtbot tests -->
 
 					<includes>
-						<include>**/AllTests.class</include>
-						<include>**/*AllTests*.class</include>
-						<include>**/*AllBotTests*.class</include>
-						<include>**/*TestSuite*.class</include>
+						<include>**/*Test*.class</include>
+						<include>**/*Test.class</include>
 					</includes>
+					<excludes>
+						<exclude>**/*Abstract*.class</exclude>
+						<exclude>**/$*.class</exclude>
```

Comment:
Running everything called *Test* = too many tests run.
Excluding *Abstract* is pointless since if they are named Abstract they should also be marked as abstract and not run at all anyways.

So if you want to have it be redundant of what the defaults are then at least let us make sure they make sense.